internal/server/server.go

```diff
@@ -0,0 +1,30 @@
+package server
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/LoaltyProgramm/to-do-app/internal/config"
+)
+
+func StartServer(cfg *config.Config) error {
+	port := cfg.Port
+	if port == "" {
+		port = "7540"
+	}
+
+	staticPath := cfg.StaticPath
+	if staticPath == "" {
+		staticPath = "../web"
```

Comment:
Команды запуска лучше исполнять из корня, так что путь тоже от корня идет
```suggestion
		staticPath = "web"
```